screens/EditSignal/index.tsx

```diff
@@ -42,21 +60,22 @@ export default function EditSignal() {
       console.error(error.message)
     },
     onSettled: async () => {
-      await fetchMySignal()
       setIsLoading(false)
     },
   })
 
   const handleSaveStatus = async () => {
     mutation.mutate()
-    saveStatus()
   }
-  useEffect(() => {
-    const removeListener = navigation.addListener("beforeRemove", () =>
-      clearStatus(),
-    )
```

Comment:
Don't we need to keep this, because once the user closes the drawer with the native scrolling, we need to clear the status as well. 

now this is current behaviour 

https://github.com/user-attachments/assets/d88d8391-273c-48d9-b94e-af7fc712acf4